docs/src/main/sphinx/language/types.rst

```diff
@@ -168,8 +168,10 @@ Example: ``TIME '01:02:03.456'``
 
 Time of day (hour, minute, second, millisecond) with a time zone.
 Values of this type are rendered using the time zone from the value.
+Time zones are expressed as the numeric UTC offset value:
 
-Example: ``TIME '01:02:03.456 America/Los_Angeles'``
+    SELECT TIME '01:02:03.456 -08:00';
+    == 1:02:03.456-08:00
```

Comment:
I am so sorry @meggpie  .. I messed up. it should be -- not ==

can you please fix